src/cf.data.pre

```diff
@@ -3076,6 +3076,82 @@ DOC_START
 			used.
 DOC_END
 
+NAME: tls_outgoing_options_for_retries
+IFDEF: USE_GNUTLS||USE_OPENSSL
+TYPE: Security::PeerContexts*
+REPETITIONS: update
+DEFAULT: none
+DEFAULT_DOC: Do not retry the same destination after ERR_SECURE_CONNECT_FAIL.
+LOC: Config.ssl_client.retriesContexts
+DOC_START
+
+	Enables same-destination retries (with new custom TLS options) after
+	ERR_SECURE_CONNECT_FAIL.
+
+		tls_outgoing_options_for_retries <option>... [if [!]<acl>...]
+
+	See tls_outgoing_options for documentation of recognized options,
+	including "ciphers=..." and "cafile=...".
+
+	This directive is consulted whenever an attempt to establish a TLS
+	connection with an origin server fails with ERR_SECURE_CONNECT_FAIL:
+
+	* If no rules are configured or if no configured rules matched, Squid does
+	  not treat ERR_SECURE_CONNECT_FAIL outcome specially (i.e. Squid may
+	  still attempt to reforward the request to a different IP address, using
+	  regular reforwarding logic).
```

Comment:
While I agree this could be improved, I don't think it's unclear or misleading to
my - and here I proxy for people with a moderate amount of understanding.
I correctly understood the target use case for this option by just reading this help
text, that's a pretty high bar to clear already.
The kind of subtleties I see here may not even belong to a configuration template,
but to a book.
I'd defer any improvements to a followup PR, but TBH even if there were none,
I would be happy